example/algorithms_evaluation/run.py

```
@@ -0,0 +1,194 @@
+		return benchmark.endswith(".aeon") or benchmark.endswith(".bnet") or benchmark.endswith(".sbml")
+
+	# Create output directory
+	BENCH_DIR_NORM = BENCH_DIR.replace("/", "_").replace(".", "")
+	OUT_DIR = BENCH_DIR_NORM + "_" + os.path.basename(SCRIPT)
+	if PARALLEL > 0:
+		OUT_DIR = OUT_DIR + "_parallel"
+	OUT_DIR = "_run_" + OUT_DIR + "_" + str(int(time.time()))
+	os.mkdir(OUT_DIR)
+
+	# Create output stats file
+	TIMES = open(OUT_DIR + "/" + BENCH_DIR_NORM + "_" + os.path.basename(SCRIPT) + "_times.csv", "w")
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)